install/update_91_92.php

```diff
@@ -1191,6 +1191,65 @@ function update91to92() {
       }
    }
 
+   // add projecttemplate
+   $migration->addField("glpi_projects", "projecttemplates_id", "integer");
```

Comment:
You must check if field does not exists yet before adding it; so the update can be played again. Samoe fo indexes.